hack/update-instance-labels.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Replace 'app.kubernetes.io/version: v0.6.x' with 'app.kubernetes.io/version: v0.7.0'
+grep -rl --exclude-dir=hack 'app.kubernetes.io/version: v0.6' ./ | xargs sed -i -E 's/app.kubernetes.io\/version: v0.6(.*)/app.kubernetes.io\/version: v0.7.0/g'
```

Comment:
we may want to exclude what hack/gen-target-tests.sh excludes which is

kfdef|gatekeeper|gcp/deployment_manager_configs|aws/infra_configs|doc|tests|hack|plugins

for example I think it will pick up files under tests